docs/sdk/core-kit/mpc-core-kit/usage.mdx

```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
   - `mnemonicToKey()` - Converts a Mnemonic to a BN.
   - `keyToMnemonic()` - Converts a hex string to a Mnemonic.
 
+## Set TSS Wallet Index
+
+#### `setTssWalletIndex(accountIndex: number): void;`
+
+Sets the TSS Wallet Index for the User's account. This is a new feature that supports key derivation, allowing users to derive multiple private keys
+from a single account for improved asset organization.
+
+#### Usage
+
+```tsx
+await coreKitInstance.setTssWalletIndex(index);
+```
+
+## Get Public Sync
+
+#### `getPublicSync()`
+
```

Comment:
what is use of this function?

Reply:
Please check once again as per the new description below.